docs/rst/spelling_wordlist.txt

```diff
@@ -14,9 +14,11 @@ eProsima
 gtest
 localhost
 multicast
+NaN
 Qos
 QoS
-Todo
+Resizable
```

Comment:
```suggestion
sesizable
```
Lower case words apply to lower and capital letter words.
Use lower case for generic words.